traversability_estimation/include/traversability_estimation/TraversabilityMap.hpp

```diff
@@ -170,14 +171,39 @@ class TraversabilityMap {
   bool readParameters();
 
   /*!
-   * Gets the traversability value of the submap defined by the polygon. Is true if the
-   * whole polygon is traversable.
+   * Gets the traversability value of the submap defined by the polygon. Is true if the whole polygon is traversable.
    * @param[in] polygon polygon that defines submap of the traversability map.
-   * @param[out] traversability traversability value of submap defined by the polygon, the traversability
+   * @param[in] computeUntraversablePolygon true if untraversable polygon within submap checked for traversability should be computed.
+   * @param[out] traversability traversability value of submap defined by the polygon, the traversability.
```

Comment:
You split the sentence accidentally. "traversability value of submap defined by the polygon, the traversability is the mean of each cell within the polygon."